<launch>
    <arg name="pupil_ip" default="localhost"/>
    <arg name="pupil_port" default="50020"/>
    <arg name="config_path" default="$(find gaze_detector)/config/remote_calibration_config.yaml"/>
    <arg name="marker_path" default="$(find gaze_detector)/config/marker.png"/>

    <node name="remote_calibration" pkg="gaze_detector" type="remote_calibration.py" respawn="false" output="screen">
        <param name="pupil_ip" value="$(arg pupil_ip)"/>
        <param name="pupil_port" value="$(arg pupil_port)"/>
        <param name="marker_path" value="$(arg marker_path)"/>
        <rosparam file="$(arg config_path)" command="load"/>
    </node>
</launch>